lib/cocoapods-search/command/search.rb

```diff
@@ -22,6 +22,7 @@ def self.options
         options += Platform.all.map do |platform|
           ["--#{platform.name.to_s}",     "Restricts the search to Pods supported on #{Platform.string_name(platform.to_sym)}"]
         end
+        options << ['--no-pager',    'Do not pipe search results into a pager']
```

Comment:
not a big deal, either works, but my thinking was that `--pager` feels weirder than `--page`